### PR TITLE
Add ipywidgets 8 notebook

### PR DESCRIPTION
**Pull request by ibdafna**
Signed-off-by: Itay Dafna <i.b.dafna@gmail.com>

### CODE REVIEW

**Comment by ibdafna** [comment]
Working through this - will finish adding the rest on Monday

File: notebooks/08.Ipywidgets_8/08.00-ipywidgets_8.ipynb [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+   "source": [
+    "#### ErrorWidget fallback when widget models or views fail."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2b912ea3-da50-4579-820e-4bdc3996d918",
+   "metadata": {},
+   "source": [
+    "#### ADd `Stacked` layout attribute"
```
Comment (jasongrout) [inline notebooks/08.Ipywidgets_8/08.00-ipywidgets_8.ipynb:248]:
```suggestion
    "#### Add `Stacked` widget"
```

The Stacked widget is a widget, not a layout attribute

**Comment by jasongrout** [comment]
https://github.com/jupyter-widgets/ipywidgets/pull/3513 has a few more changes (ready to review...)

**Comment by jasongrout** [comment]
The FileUpload widget changed significantly, and probably should be called out.

**Comment by ibdafna** [comment]
@jasongrout added FileUploader content. Let me know if it looks good enough for merging

**Comment by mwcraig** [comment]
Looks excellent, I'm going to merge it!

**Pull request merged**